app/controllers/welcome_controller.rb

```diff
@@ -1,4 +1,16 @@
 class WelcomeController < ApplicationController
   def index
+
+    food_name = params[:food_name]
```

Comment:
Not sure where this `food_name` param is coming from, since the form on the `welcome/index.html.erb` file is sending in a param called `:q`. 

I am wondering if you tried doing a `binding.pry` here and checking out your `params`?